### PR TITLE
add examples to create dns record

### PR DESCRIPTION
**Pull request by mftzk**
- [x] I understand that this repository is auto-generated and my pull request may not be merged

## Changes being requested
only add example for create dns recored inside zone

## Additional context & links
no other context, just make it easy for newbie using this library

### CODE REVIEW

File: examples/dns/record.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from cloudflare import Cloudflare
+
+
+# define your variable here
+
+cloudflare_token = "your-cloudflare-token"
+cloudflare_zone_id = "zone"
+alias_name = "www.mydns.com"
+cannonical_name = "cname.example.com"
```
Comment (jacobbednarz) [inline examples/dns/record.py:9]: let's move these inline to the kwargs as using variables doesn't give us anything extra here.

File: examples/dns/record.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+# build cloudflare driver here
+client = Cloudflare(api_token=cloudflare_token)
```
Comment (jacobbednarz) [inline examples/dns/record.py:12]: we shouldn't need to define `api_token` config here. if the value is defined in the environment, we pick that up.
```suggestion
client = Cloudflare()
```

File: examples/dns/record.py (hunk 1 above)
Comment (jacobbednarz) [reply examples/dns/record.py:9]: we can also use a similar pattern to https://github.com/cloudflare/cloudflare-python/blob/d04dfee45618904afb9569b0202758b8d6b63801/examples/workers/ai/demo.py#L6-L8 for the `zone_id` values.

File: examples/dns/record.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+from cloudflare import Cloudflare
+
+
+# define your variable here
```
Comment (jacobbednarz) [inline examples/dns/record.py:4]:
```suggestion
```

File: examples/dns/record.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+cloudflare_token = "your-cloudflare-token"
+cloudflare_zone_id = "zone"
+alias_name = "www.mydns.com"
+cannonical_name = "cname.example.com"
+
+# build cloudflare driver here
```
Comment (jacobbednarz) [inline examples/dns/record.py:11]:
```suggestion
```

File: examples/dns/record.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+dns = client.dns.records.create(
+    zone_id=cloudflare_zone_id, 
+    type='CNAME', 
+    name=alias_name, 
+    content=cannonical_name,
+    proxied=False
+    )
+
+# print output
+print(dns)
```
Comment (jacobbednarz) [inline examples/dns/record.py:26]: let's avoid overlapping `dns` here as it's part of the package name and the types will be confused if someone opts into them.

```suggestion
record = client.dns.records.create(
    zone_id=cloudflare_zone_id, 
    type='CNAME', 
    name=alias_name, 
    content=cannonical_name,
    proxied=False
    )

# print output
print(record)
```

File: examples/dns/record.py (hunk 2 above)
Comment (mftzk) [reply examples/dns/record.py:10]: oh got it

**Pull request merged**